src/app/containers/Navigation/index.jsx

```diff
@@ -10,7 +10,27 @@ import { RequestContext } from '#contexts/RequestContext';
 import Canonical from './index.canonical';
 import Amp from './index.amp';
 
-const Navigation = () => {
+const renderListItems = (Li, navigation, script, currentPage, service, dir) =>
+  navigation.map((item, index) => {
+    const { title, url } = item;
+    const active = index === 0;
+
+    return (
+      <Li
+        key={title}
+        url={url}
+        script={script}
+        active={active}
+        currentPageText={currentPage}
+        service={service}
+        dir={dir}
+      >
+        {title}
+      </Li>
+    );
+  });
+
+const NavigationContainer = () => {
   const { platform } = useContext(RequestContext);
   const isAmp = platform === 'amp';
```

Comment:
I have created an issue to address this check. I think this is spread all over our containers while we have this already taken care of in the `RequestContext` https://github.com/bbc/simorgh/issues/4970.

Reply:
Thanks - that's a good idea.